c2corg_ui/templates/outing/index.html

```diff
@@ -27,7 +27,7 @@
   <div class="documents-nav-buttons text-center">
     <button class="btn btn-default show-documents-filters-phone" translate>Filters</button>
     ${add_doctype_selector()}
-    <button class="btn btn-default" protected-url-btn url="${request.route_path('outings_add')}">
+    <button class="btn btn-default orange-btn" protected-url-btn url="${request.route_path('outings_add')}">
```

Comment:
What about adding a tooltip on such buttons as well? Or even a real title?
I'm afraid many people won't understand what this button is for.